src/users/models.py

```diff
@@ -33,7 +33,7 @@ class Mentor(models.Model):
     major = models.ForeignKey(Major, null=True, on_delete=models.SET_NULL)
     bio = models.CharField(max_length=5000, null=False, blank=True, default='')
     active = models.BooleanField(default=True)
-    gpa = models.DecimalField(default=0.00, max_digits=4, decimal_places=2)
+    gpa = models.DecimalField(blank=True, default=0.00, max_digits=4, decimal_places=2)
```

Comment:
any reason why?

Reply:
Ram was getting an error when using mentors/me/, and the error was the same as when I forgot to allow classes and clubs to be blank. I couldn't replicate the error, so I just put it in to be safe.

Reply:
you want null=True. but I highly suspect Ram didn't run a migration

Reply:
So take it out, or switch it to null=True?

Reply:
I think you can take it out. We already have a default

Reply:
we shouldn't add random code to fix a problem if we're not sure it even fixes it